software-engineering/software-engineering.md

```diff
@@ -0,0 +1,107 @@
+# Software Engineering
+
+The Open Systems Pharmacology Suite (OSP Suite) is the open-source platform. The OSP Suite is the result of over 15 years of effort from experts in software development, database management, mathematics, biology, physiology, pharmacokinetics, pharmacology, and systems biology. This multi-million dollar platform was made open-source in early 2017 to accelerate the development and application of PBPK models and systems modeling in the drug development process.
+
+## Source Control
+The OSP Suite uses GitHub as a source control platform.
+
+![The OSPSuite Github Organization](../assets/images/osp-github.png)
+
+GitHub is a web-based hosting service for version control using Git. It offers all of the distributed version control and source code management (SCM) functionality of Git as well as adding its own features. It provides access control and several collaboration features such as bug tracking, feature requests, task management, and wikis for every project.
+
+GitHub offers plans for both private repositories and free accounts, which are commonly used to host open-source software projects. All Repositories in the scope of the OSP Suite are public. As of January 2023, GitHub reports having over 100 million users and more than 420 million repositories (including at least 28 million public repositories), making it the largest host of source code in the world.
+
+In addition to source code, GitHub supports the following formats and features:
+* Documentation, including automatically rendered README files in a variety of Markdown-like file formats
+* Issue tracking (including feature requests) with labels, milestones, assignees and a search engine
+* Wikis
+* Pull requests with code review and comments
+* Commits history
+* Graphs: pulse, contributors, commits, code frequency, punch card, network, members
+* Email notifications and the option to subscribe someone to notifications by @ mentioning them
+* GitHub Pages: small websites can be hosted from public repositories on GitHub. This is what we have done for the OSP Suite where the site http://www.open-systems-pharmacology.org/ is a user friendly gateway to the OSP Platform
+* PDF document viewer
```

Comment:
add also:
* [GitHub Codespaces](https://en.wikipedia.org/wiki/GitHub_Codespaces), an online Integrated Development Environment (IDE) providing users with a virtual machine intended to be a work environment to build and test code
* [Discussions](https://docs.github.com/en/discussions)
* Security Alerts of known [Common Vulnerabilities and Exposures](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures) in different packages